TrainingRNExpress/src/app/components/Book/styles.js

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+
+import { GREY, WHITE, DARK_GREY } from '../../../constants';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: WHITE,
+    borderRadius: 10,
+    borderBottomWidth: 1,
+    borderBottomColor: GREY,
+    display: 'flex',
+    flexDirection: 'row',
+    padding: 10
+  },
+  icon: {
+    marginRight: 20,
+    height: 50,
+    width: 50
```

Comment:
I'd extract this `50` in a constant within this file since this one always behaves as a square